libbeat/docs/security/users.asciidoc

```diff
@@ -259,10 +260,24 @@ ifeval::["{beatname_lc}"=="filebeat"]
 |Check for ingest pipelines used by modules. Needed when using modules.
 endif::[]
 
+ifeval::["{beatname_lc}"=="winlogbeat"]
+|Cluster
+|`read_pipeline`
+|Check for ingest pipelines used by {beatname_uc}.
+endif::[]
+
 |Index
 |`create_doc` on +{beat_default_index_prefix}-*+ indices
 |Write events into {es}
+
+|Index
+|`auto_configure` on +{beat_default_index_prefix}-*+ indices
+|Update the datastream mapping. Consider either disabling entirely or adding the
+rule `-{beat_default_index_prefix}-*` to the cluster settings
```

Comment:
```suggestion
rule `+-{beat_default_index_prefix}-*+` to the cluster settings
```

This would render like the following:

![Screenshot 2024-06-11 at 10 00 15 AM](https://github.com/elastic/beats/assets/41695641/cd698cdc-f117-4a14-b64b-c8d94a413116)

Reply:
> I am not entirely sure on how to fix that. I think:
> -{beat_default_index_prefix}-*
> Should actually become:
> +-{beat_default_index_prefix}-*+

@herrBez  The suggestion above will add those `+` signs, if I understand correctly that that's what should be changed.

Reply:
Sorry, I want it to render to (e.g., `filebeat-*`). How can I achieve that :) ASCIIDoc and variables are not my forte.

Reply:
Ah, I'm sorry! I misunderstood. I don't work with Beats much so I missed that `{beat_default_index_prefix}` is a variable. Variables won't be interpolated when they're inside backticks. So, I think the best we can do here is to take them out, like this:

> rule {beat_default_index_prefix}-* to the cluster settings

![Screenshot 2024-06-13 at 9 46 42 AM](https://github.com/elastic/beats/assets/41695641/d3511778-ee3e-4c6f-ba3a-121eaf1d6e3a)

Reply:
Thank you, I guess I got it (it should be the same as on the one on the right). How are you doing the rendering ? (To avoid this problem in the future :) )

Reply:
My pleasure Mirko! It looks good now.

To do the local rendering, you can:
 - clone the [elastic/docs](https://github.com/elastic/docs) repo
 - have this [doc_build_aliases](https://github.com/elastic/docs/blob/master/doc_build_aliases.sh) file in the root of your local github directory

Then, use any of the aliases in that file with `--open`. For example:
  - `docbldfb --open` - generates the Filebeat docs
  - `docbldim --open` - generates the Fleet & Agent Guide
  
Most docs builds take just a minute or two. The largest, for Elasticsearch docs (dodbldes --open) takes about five minutes.